json-endpoints/class.wpcom-json-api-edit-media-v1-2-endpoint.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
```

Comment:
Yeah, it's confusing.  tl;dr this `@hide-in-jetpack` tells a script that we use to hide this block from Jetpack and it's safe to remove from this PR.  

Anyway, it's fine to leave it in too. Not a blocker.